modules/rest_api/controllers/services/rest.php

```diff
@@ -1044,16 +1044,40 @@ private function checkAllowedResource($proj_id, $resourceName) {
       ['caption' => 'Comment', 'field' => 'occurrence.occurrence_remarks'],
       ['caption' => 'Sample comment', 'field' => 'event.event_remarks'],
       ['caption' => 'Images', 'field' => '#occurrence_media#'],
-      ['caption' => 'Input on date', 'field' => 'metadata.created_on'], // Format ??
-      ['caption' => 'Last edited on date', 'field' => 'metadata.updated_on'], // Format ??
-      ['caption' => 'Verification status 1', 'field' => 'identification.verification_status'],
-      ['caption' => 'Verification status 2', 'field' => '#null_if_zero:identification.verification_substatus#'],
-      ['caption' => 'Query', 'field' => 'identification.query'],
+      ['caption' => 'Input on date', 'field' => '#datetime:metadata.created_on:d/m/Y H\:i#'],
+      ['caption' => 'Last edited on date', 'field' => '#datetime:metadata.updated_on:d/m/Y H\:i#'],
+      ['caption' => 'Verification status 1', 'field' => '#verification_status:standard#'],
+      ['caption' => 'Verification status 2', 'field' => '#verification_substatus:standard#'],
+      ['caption' => 'Query', 'field' => '#query:standard#'],
       ['caption' => 'Verifier', 'field' => 'identification.verifier.name'],
-      ['caption' => 'Verified on', 'field' => 'identification.verified_on'],
+      ['caption' => 'Verified on', 'field' => '#datetime:identification.verified_on:d/m/Y H\:i#'],
+      ['caption' => 'Licence', 'field' => 'metadata.licence_code'],
+      ['caption' => 'Automated checks', 'field' => 'identification.auto_checks.result'], 
+    ],
+    "mapmate" => [
+      ['caption' => 'Taxon', 'field' => 'taxon.accepted_name'],
+      ['caption' => 'Site', 'field' => 'location.verbatim_locality'],
+      ['caption' => 'Gridref', 'field' => 'location.output_sref'],
+      ['caption' => 'VC', 'field' => '#vc:mapmate#'],
+      ['caption' => 'Recorder', 'field' => 'event.recorded_by'],
+      ['caption' => 'Determiner', 'field' => 'identification.identified_by'],
+      ['caption' => 'Date', 'field' => '#record_date:mapmate#'],
```

Comment:
Why have we added #record_date# as an extra special field when we already have #event_date#? The mapmate format specified could be added to #event_date#?

Reply:
This occurred to me when I was updating the documentation yesterday! I agree and will change.

Reply:
Completed. I came across an oddity when doing this. Under some circumstances, the existing EventDate handler would return format dates as yyyy-mm-dd (I noticed in date ranges) and sometimes as dd/mm/yyyy. So I also added some reformatting of the former to the latter.